actors/v2/miner/parse.go

```diff
@@ -150,6 +150,9 @@ func (m *Miner) Parse(_ context.Context, network string, height int64, txType st
 	case parser.MethodProveCommitSectorsNI:
 		resp, err := m.ProveCommitSectorsNI(network, height, msg.Params)
 		return resp, nil, err
+	case parser.MethodInitialPledgeExported, parser.MethodInitialPledge:
+		resp, err := m.InitialPledgeExported(network, height, msg.Params)
+		return resp, nil, err
```

Comment:
Was this a missing method? wow! Makes me wonder why it was not caught by our tests. I remember you said there is a test for missing methods, right?

Reply:
Wow! interesting...